sycl/plugins/unified_runtime/CMakeLists.txt

```
@@ -3,8 +3,8 @@
 if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_DIR)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG e6343f4cca9a37b17bc63f3a81968ac3f486be8a)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/npmiller/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG b4425bc19fb9eaa445247c2179563a1f7e9356cc)
```

Once the conflicts have been resolved this should be updated to point at:

```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 3a3aae387eaa8aa072ff310b94df83c9d1a33a0b
  # Merge: 614e6d0 b4425bc
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Mon Oct 23 14:49:01 2023 +0100
  #     Merge pull request #945 from npmiller/fix-priority
  #     [CUDA] Fix queue creation with native handle
  set(UNIFIED_RUNTIME_TAG 3a3aae387eaa8aa072ff310b94df83c9d1a33a0b)
```